test/integration/feeds.test.ts

```diff
@@ -13,13 +13,30 @@ describe(`${Feeds.name}`, () => {
 
     const parameters = {
       FeedSubmissionId: '51793018437',
+      format: 'xml' as 'xml',
```

Comment:
This is weird, why not just do:
```ts
const parameters: GetFeedSubmissionResultParameters = {
  FeedSubmissionId: '51793018437',
  format: 'xml'
}
```

Reply:
Yep, I was being lazy. Thought it wouldn't matter since it was tests, but I should've typed it out properly from the start. Thanks for catching it! :+1: